contrib/ros/robots/robot.py

```diff
@@ -0,0 +1,33 @@
+"""
+robot Interface
+"""
+
+
+class Robot(object):
+    def move_to_start_position(self):
+        """
+        User uses this to make the robot move to a start position at the beginning of every training episode.
+        """
+        raise NotImplementedError
+
+    def set_command(self, commands):
```

Comment:
Does this set a command or send a command? What is the type of the object command?

Reply:
Shouldn't `command` be in terms of the action space?